### DISCUSSION TITLE
Use all the files in a project as context

### DISCUSSION THREAD

**Discussion by louib** [CopilotC-Nvim/CopilotChat.nvim#160] [Q&A]
From what I understand, only the current buffer is used as context when interacting in a generic way with copilot (`:CopilotChatOpen`). This is at least my impression after asking a few questions related to a library I am working on. Is that expected, or am I missing some configuration?

**Comment by deathbeam** [comment]
By default no context is used. Canary supports `@buffer` and `@buffers` for specifying it, all workspace files are not supported atm and its non-trivial to implement (adds python dependency back at minimum), but its in roadmap at least and I have working workspace watcher already implemented in python, but its big dependency and would require for users to run it in docker or just locally so not sure if I want to PR integration for it or not

**Comment by louib** [comment]
@deathbeam Thanks a lot for the context, I'll have a look at the canary branch.
